src/main/java/com/github/dockerjava/jaxrs/filter/ResponseStatusExceptionFilter.java

```diff
@@ -80,9 +80,12 @@ private String getBodyAsMessage(ClientResponseContext responseContext) {
 
                 if (MediaType.APPLICATION_JSON_TYPE.equals(mediaType)) {
                     ObjectMapper mapper = new ObjectMapper();
-                    JsonNode node = mapper.readTree(entityStream).get("message");
+                    JsonNode node = mapper.readTree(message);
```

Comment:
maybe surround with try? Parsing here is optional and shouldn't fail response exceptions

Reply:
Done